lib/PrintAsClang/PrintAsClang.cpp

```diff
@@ -53,6 +53,8 @@ static void emitObjCConditional(raw_ostream &out,
 static void writePrologue(raw_ostream &out, ASTContext &ctx,
                           StringRef macroGuard) {
 
+  // NOTE: If you add macros to this prologue, also add them to the `compatibility_symbols` file!
```

Comment:
Can you add the path to the file in the comment?

Also, how does this generate the compatibility symbols below? We really need some sort of test to make sure that if the compatibility symbols change that the file below is updated.

It could be as simple as creating a test that runs PrintAsClang and diffs it against that file... even though it would be better to just actually generate it as part of the build... your thoughts?

Reply:
It does not generate this, yet. I put the PR up to try and make sure that i was setting up the installation commands correctly.

I wasn't sure the best way to find the file from a lit test, hence my question earlier:

> Is it possible to reference this file from a lit test? I would love to be able to verify that the generated compatibility header actually matches the list of symbols in the installed file, but i'm not sure the best way to approach it.